src/mame/drivers/mc10.cpp

```diff
@@ -26,123 +24,105 @@
 
 #include "formats/coco_cas.h"
 
-
-namespace
-{
-
 /***************************************************************************
     TYPE DEFINITIONS
 ***************************************************************************/
```

Comment:
The type declarations are entirely inside the anonymous namespace (along with everything else besides the driver definition itself) – there’s no reason for the heading comment to be outside the namespace.